### PR TITLE
Backport on focus

### PR DESCRIPTION
**Pull request by SteamRiC**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have used ESLint & Prettier to follow the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/IntlTelInputApp.js [hunk 1]
```diff
@@ -222,6 +222,9 @@ class IntlTelInputApp extends Component {
   }
 
   setTelRef = (ref) => {
+    if (typeof this.props.telInputRefCallback === 'function') {
+      this.props.telInputRefCallback(ref);
+    }
     this.tel = ref;
```
Comment (ostap0207) [inline src/components/IntlTelInputApp.js]: There is `this.tel`, this wasn't enough?

**Review by ostap0207 — APPROVED** [review]

File: src/components/IntlTelInputApp.js (hunk 1 above)
Comment (ostap0207) [reply src/components/IntlTelInputApp.js]: I imagine that the answer is no :)

Comment (SteamRiC) [reply src/components/IntlTelInputApp.js]: good catch. it actually allows to get the number input ref from IntlTelInputApp, when just passing a ref to it. so the extra prop is not required

Comment (SteamRiC) [reply src/components/IntlTelInputApp.js]: removed the commit

**Comment by SteamRiC** [comment]
@ostap0207 good to merge? Apparently siim is dead today and I need this merged for the other PR in omnibrowse-app

**Comment by ostap0207** [comment]
Yep

**Pull request merged**